examples/2d/BoxAroundCircle8.mesh

```diff
@@ -0,0 +1,345 @@
+          56          96          40           8
```

Comment:
For now I am working on adding to the docs a section about the unstructured mesh format. Then I think we can point to the docs rather than having to alter the raw mesh file data with comments

Reply:
I added a draft of the unstructured mesh documentation with a figure into the `docs` folder